index.html

```diff
@@ -22,18 +22,117 @@ <h2 class="text-center">Explore</h2>
   </div>
 </div>
 
+<style>
+
```

Comment:
You see these spaces. When you lint a code these type of problems do not come. Try adding this in site [reference](https://github.com/Gizra/KnowledgeBase/wiki/Add-grunt-scss-lint-to-your-jekyll-site.) and then lint the code and update your PR.